test/component-test.js

```diff
@@ -59,7 +59,20 @@ describe('component creation', () => {
     });
 
     context('node is not scoped', () => {
-      it('recursively creates elements with data');
+      it('recursively creates elements with data', () => {
+        var chattr = new Attribute('data-prop', 'wat');
```

Comment:
Aw c'mon Mr. Rubyist, we don't need to abbreviate, right? ;-) I'd say a `childAttr` is a bit clearer. `Attr` is a common enough abbreviation that I don't think there's any value to expanding it, though.